doc/reference/reference_lua/fiber.rst

```diff
@@ -779,6 +785,46 @@ recommended.
             - 1448466351270762
             ...
 
+.. _fiber-clock:
+
+.. function:: clock()
+
+    Get the monotonic time in seconds. It is better to use ``fiber.clock()`` for
+    calculating timeouts instead of ``fiber.time()`` because ``fiber.time()``
```

Comment:
maybe add a link here?

...instead of :ref:`fiber.time() <fiber-time>` because ...

Reply:
fixed in 1c92371